### PR TITLE
Feature/modal size

### LINKED ISSUE #123: Movie Cards do not fill screen on mobile
## Expected Behavior
The movie card should fill the screen on mobile devices.

## Current Behavior
There is padding on the left and right of the movie card.

## Possible Solution
Remove padding around movie card for mobile.

## Steps to Reproduce
1. Navigate to groa.us/explore (on mobile)
2. Select a movie poster to open the movie card.
3. Observe the padding to the right and left of the movie card.

## Context (Environment)

This issue primarily hinders the ability of the movie trailers to preview entirely within the movie card on mobile. On desktop we don't want the movie card to take up the entire screen.

### PR DESCRIPTION
**Pull request by Steven-matos**
# Description

Fixes the modal size on mobile.

Fixes #123 

Please add a list of what someone reviewing the PR
should be expecting from this PR.
This will include the

- Fixes the width of the movie modal to fit the phone screen.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [x] Complete, tested, ready to review and merge

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/movies/MovieCard.js [hunk 1]
```diff
@@ -27,10 +27,10 @@ import CloseIcon from "@material-ui/icons/Close";
...
 //menu expander
-import ExpansionPanel from '@material-ui/core/ExpansionPanel';
-import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
-import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
-import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+// import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+// import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+// import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+// import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 //import for button group
```
Comment (msinnema33) [inline src/components/movies/MovieCard.js:34]: Are these needed?  If not, remove.

File: src/components/movies/MovieCard.js [hunk 2]
```diff
@@ -356,18 +356,18 @@ function MovieCard({
   const anchorRef = React.useRef(null);
   const [selectedIndex, setSelectedIndex] = React.useState(1);
 
-  const handleClickServiceProvider = () => {
-    console.info(`You clicked ${serviceProvider[selectedIndex]}`);
-  };
+  // const handleClickServiceProvider = () => {
+  //   console.info(`You clicked ${serviceProvider[selectedIndex]}`);
+  // };
```
Comment (msinnema33) [inline src/components/movies/MovieCard.js:362]: remove?

File: src/components/movies/MovieCard.js [hunk 3]
```diff
@@ -356,18 +356,18 @@ function MovieCard({
...
     setSelectedIndex(index);
     setOpen(false);
   };
 
-  const handleToggle = () => {
-    setOpen((prevOpen) => !prevOpen);
-  };
+  // const handleToggle = () => {
+  //   setOpen((prevOpen) => !prevOpen);
+  // };
```
Comment (msinnema33) [inline src/components/movies/MovieCard.js:371]: remove?

File: src/components/movies/MovieCard.js [hunk 4]
```diff
@@ -551,6 +551,7 @@ function MovieCard({
                     size="large"
                     precision={0.5}
                     emptyIcon={<StarBorderIcon fontSize="inherit" />}
+                    // eslint-disable-next-line react/jsx-no-duplicate-props
```
Comment (msinnema33) [inline src/components/movies/MovieCard.js:554]: remove?

File: src/components/movies/MovieCard.js [hunk 5]
```diff
@@ -570,6 +571,7 @@ function MovieCard({
...
                     emptyIcon={
```
Comment (msinnema33) [inline src/components/movies/MovieCard.js:575]: remove?

File: src/utils/widthFinder.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 const widthFinder = (screenWidth) => {
-  if (parseInt(screenWidth) < 400) {
+  console.log("This is the size: ", screenWidth);
+  if (parseInt(screenWidth) < 450) {
```
Comment (msinnema33) [inline src/utils/widthFinder.js:3]: remove console log (line 2)

Comment (Steven-matos) [reply src/utils/widthFinder.js:2]: good catch removed

File: src/components/movies/MovieCard.js (hunk 5 above)
Comment (Steven-matos) [reply src/components/movies/MovieCard.js]: this is in place to stop a duplicate prop error in VS code

File: src/components/movies/MovieCard.js (hunk 4 above)
Comment (Steven-matos) [reply src/components/movies/MovieCard.js]: this is in place to stop a duplicate prop error in VS code

File: src/components/movies/MovieCard.js (hunk 3 above)
Comment (Steven-matos) [reply src/components/movies/MovieCard.js:375]: will be used for service provider at a later date

File: src/components/movies/MovieCard.js (hunk 2 above)
Comment (Steven-matos) [reply src/components/movies/MovieCard.js:366]: will be used for service provider at a later date

File: src/components/movies/MovieCard.js (hunk 1 above)
Comment (Steven-matos) [reply src/components/movies/MovieCard.js:34]: will be used for service provider at a later date

**Pull request merged**